<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferences</class>
 <widget class="QDialog" name="preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PiCamera Preferences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>630</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>371</width>
     <height>591</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="General">
    <attribute name="title">
     <string>&amp;General</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="setPhotoPath">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>99</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;PibotoLt&quot;;</string>
      </property>
      <property name="text">
       <string>Photos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setVideoPath">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>99</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;PibotoLt&quot;;</string>
      </property>
      <property name="text">
       <string>Videos</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setFilePath">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>99</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;PibotoLt&quot;;</string>
      </property>
      <property name="text">
       <string>Files</string>
      </property>
     </widget>
     <widget class="QLabel" name="defaultPhotoPath">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color : blue</string>
      </property>
      <property name="text">
       <string>/home/pi/Pictures</string>
      </property>
     </widget>
     <widget class="QLabel" name="defaultVideoPath">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color : blue</string>
      </property>
      <property name="text">
       <string>/home/pi/Videos</string>
      </property>
     </widget>
     <widget class="QLabel" name="defaultFilePath">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color : blue</string>
      </property>
      <property name="text">
       <string>/home/pi/Documents</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Set default directories </string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>351</width>
       <height>101</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Photo capture format&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Video capture format&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>268</x>
        <y>10</y>
        <width>71</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 italic 10pt &quot;PibotoLt&quot;;
font: 11pt &quot;PibotoLt&quot;;</string>
      </property>
      <property name="text">
       <string>parms...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>268</x>
        <y>50</y>
        <width>71</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;PibotoLt&quot;;</string>
      </property>
      <property name="text">
       <string>parms...</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>83</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>83</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>351</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Timestamp format&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>308</x>
        <y>10</y>
        <width>31</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;PibotoLt&quot;;
font: 75 11pt &quot;PibotoLt&quot;;</string>
      </property>
      <property name="text">
       <string>?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>271</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Include timestamp in photo name</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>261</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Include timestamp in video name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Sample timestamp:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>161</width>
        <height>32</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>241</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Photo/Video capture formats</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>68</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>&amp;Interface</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>&amp;Other</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setPhotoPath</sender>
   <signal>clicked()</signal>
   <receiver>preferences</receiver>
   <slot>getDefaultPhotoPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setVideoPath</sender>
   <signal>clicked()</signal>
   <receiver>preferences</receiver>
   <slot>getDefaultVideoPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setFilePath</sender>
   <signal>clicked()</signal>
   <receiver>preferences</receiver>
   <slot>getDefaultFilePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getDefaultPhotoPath()</slot>
  <slot>getDefaultVideoPath()</slot>
  <slot>getDefaultFilePath()</slot>
 </slots>
</ui>
